README.md

```diff
@@ -67,9 +67,9 @@ None.
 |----------|-------------|---------|----------|
 | geoip2\_database\_directory | The directory in which to store the database files. | `/usr/local/share/GeoIP/` | No |
 | geoip2\_geoipupdate\_auto\_update | Whether to configure automatic updates when `geoipupdate` is installed. | `true` | No |
-| geoip2\_geoipupdate\_service\_name | The name to use for the `geoipupdate` SystemD service and timer. | `geoipupdate` | No |
-| geoip2\_geoipupdate\_service\_timer\_on\_calendar | The calendar expression to use for the `geoipupdate` SystemD timer. | `Wed,Sat America/New_York` | No |
-| geoip2\_geoipupdate\_service\_timer\_randomized\_delay\_sec | The randomized delay in seconds to use for the `geoipupdate` SystemD timer. | `3h` | No |
+| geoip2\_geoipupdate\_service\_name | The name to use for the `geoipupdate` systemd service and timer. | `geoipupdate` | No |
+| geoip2\_geoipupdate\_service\_timer\_on\_calendar | The calendar expression to use for the `geoipupdate` systemd timer. | `Wed,Sat America/New_York` | No |
+| geoip2\_geoipupdate\_service\_timer\_randomized\_delay\_sec | The `systemd.time` time span value that represents the randomized delay in seconds to use for the `geoipupdate` systemd timer. | `3h` | No |
```

Comment:
I still feel like the mention of seconds is very misleading here:
```suggestion
| geoip2\_geoipupdate\_service\_timer\_randomized\_delay | The `systemd.time` time span value that represents the randomized delay to use for the `geoipupdate` systemd timer. | `3h` | No |
```

Reply:
I agree with @jsf9k here.  If you really want to, you could add a clause along the lines of "if there's no time unit, then it means seconds", but I don't think it's totally necessary.

Reply:
Please see [450b394](https://github.com/cisagov/ansible-role-geoip2/pull/51/commits/450b394807a674789207cfc2951c2d16a59c19e1). I do not want to burn time trying to document the peculiarities of systemd's functionality, so I would rather just reference the options being populated and leave it to the reader.